packages/eslint-plugin/src/rules/no-unsafe-assignment.ts

```diff
@@ -37,17 +38,18 @@ export default createRule({
       requiresTypeChecking: true,
     },
     messages: {
-      anyAssignment: 'Unsafe assignment of an `any` value.',
+      anyAssignment: 'Unsafe assignment of an {{sender}} typed value.',
       anyAssignmentThis: [
-        'Unsafe assignment of an `any` value. `this` is typed as `any`.',
+        'Unsafe assignment of an {{sender}}  value. `this` is typed as `any`.',
```

Comment:
[Bug] Looks like a bad find-and-replace, two spaces? Here and elsewhere:

```suggestion
        'Unsafe assignment of an {{sender}} value. `this` is typed as `any`.',
```

Reply:
Done.